### PR TITLE
[Kotlin CleanUp] AbstractFlashcardViewerCommandTest

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin CleanUp of AbstractFlashcardViewerCommandTest.kt

## How Has This Been Tested?
Tested on a physical device

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerCommandTest.kt [hunk 1]
```diff
@@ -167,26 +163,25 @@ class AbstractFlashcardViewerCommandTest : RobolectricTest() {
...
 
         viewer.executeCommand(command)
         viewer.executeCommand(command)
 
         assertThat(command.toString(), viewer.lastFlag, equalTo(FLAG_NONE))
     }
 
-    @KotlinCleanup("rename the getter method to createViewer(), remove val")
-    private val viewer: CommandTestCardViewer
+    private val createViewer: CommandTestCardViewer
```
Comment (oakkitten) [inline AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerCommandTest.kt:174]: This should be a `fun` not a `val`

Comment (Prince-kushwaha) [reply AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerCommandTest.kt:174]: i do not understand what i have to do ?

Comment (oakkitten) [reply AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerCommandTest.kt:174]: Replace `private val createViewer: ...` with `private fun createViewer(): ...` and change the calling code accordingly. It is an action, and the name is a verb phrase (as in “I want to create a viewer”). A function fits much better here.

Comment (Prince-kushwaha) [reply AnkiDroid/src/test/java/com/ichi2/anki/AbstractFlashcardViewerCommandTest.kt]: ok  thanks for help

**Review by lukstbit — APPROVED** [review]

**Pull request merged**